src/Core/Engine/Renderer/API/OpenGL/FBO.hpp

```
@@ -7,6 +7,9 @@ class FBO final
 {
     GLuint m_ColorTexture;
     GLuint m_RBO;
+
+    double m_Width = 1.f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)